src/components/entrybutton/EntryButton.tsx

```diff
@@ -4,15 +4,16 @@ import * as Styled from "./EntryButton.styled";
 
 interface EntryButtonProps {
   label: string;
-  type?: "Primary" | "Secondary";
+  type?: "Add" | "Cancel";
+  width?: number;
 }
-function EntryButton({ label, type = "Primary" }: EntryButtonProps) {
+function EntryButton({ label, type = "Add", width = 300 }: EntryButtonProps) {
```

Comment:
Just a happy little concept: Having a type refer to the action is nice, since it communicates that this sort of button action (read: styling) will always perform that sort of action and helps sustain consistency